Views/home.stencil

```diff
@@ -13,16 +13,58 @@
     <p>Hack Cambridge</p>
   </section>
   <section class="LandingUpdateFeed">
-    <div class="LandingUpdateFeed__item">
-      <div class="PostCard">
-        <span class="PostCard__label">Workshop</span>
-        <h3 class="PostCard__title">Conquering Linux</h3>
+    <div><h1 class="LandingUpdateFeed__title Text--sectionHeading">Updates</h1></div>
+    <div class="LandingUpdateFeed__postContainer">
+      <div class="LandingUpdateFeed__item">
+        <div class="PostCard">
+          <div class="PostCard__photoBackground" style="background-image: url('/static/images/workshop.jpg')"></div>
```

Comment:
This is going to become much more exciting when _everything becomes data_ with a DSL (styles can be dictionaries!). The [CSS `attr()`](https://developer.mozilla.org/en/docs/Web/CSS/attr)  function will give you what you desire but proper support hasn't landed anywhere.

While they won't clean up your templates (DSLs will 👹), these kinds of attributes (including your colours) are a pretty perfect case for [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*).